src/main/java/com/example/javacohort3/ZipCodeBank/controllers/AccountController.java

```diff
@@ -48,7 +48,10 @@
     // Get accounts by customer ID
     @RequestMapping("/customers/{customerId}/accounts")
     public ResponseEntity<?> getAllAccountsByCustomerId(@PathVariable Long customerId) {
+
         customerService.verifyCustomer(customerId);
+        accountService.verifyCustomerById(customerId);
+
         ArrayList<Account> accounts = (ArrayList<Account>) accountService.getAccountsByCustomerId(customerId);
 
         accountService.verifyAccountById(new Long(accounts.size()));
```

Comment:
*why you using size method to verify a account?*

Reply:
Ok @imowbray97  It was just explained to me by @aermias, basically, since each account has an auto generated id, 
Then if there is only one account in the accounts arrayList it would automatically have an id one. As such the verify method would be checking the id that way.

Reply:
Also @imowbray97 if there were multiple accounts in existence but none of those accounts belonged to the customer we were checking for then that customer would not have any accounts in the arrayList so there would be no id that matches with a size 0 which is how it would know to throw the exception